src/System.Threading.ThreadPool/src/project.json

```diff
@@ -1,16 +1,16 @@
 {
   "frameworks": {
-    "netstandard1.3": {
+    "netstandard1.7": {
       "dependencies": {
         "Microsoft.TargetingPack.Private.CoreCLR": "1.2.0-beta-24617-03"
       },
       "imports": [
```

Comment:
yup, this can be removed.

---
In reply to: [84001371](https://github.com/dotnet/corefx/pull/12773#discussion_r84001371) [](ancestors = 84001371)